Makefile

```diff
@@ -17,7 +17,10 @@ KIND_CLUSTER_NAME ?= containerd-wasm
 .PHONY: build
 build:
 	cargo build -p containerd-shim-wasm --features generate_bindings $(RELEASE_FLAG)
-	cargo build -p containerd-shim-wasm --features libcontainer $(RELEASE_FLAG)
+	# compiling against libcontainer's default features (dependency on libseccomp)
+	cargo build -p containerd-shim-wasm --features libcontainer_default $(RELEASE_FLAG)
+	# compiling against libcontainer's cgroups v2 feature (no dependency on libseccomp)
+	cargo build -p containerd-shim-wasm --features cgroupsv2 $(RELEASE_FLAG)
```

Comment:
do we need to build each Indvidual feature or can we use something like `--all-features`?  I've seen that in other projects and I am not sure what the best thing to do here. Otherwise it seems like we need to do every combination that is exposed.

Reply:
--all-features are just additively adding all the features, and that's good to build. But we also want to make sure each individual feature is buildable. 

Unrelated: I am thinking about writing an integration tests to `grep` the output of `cargo update` on different features to make sure `libseccomp` is adding/deleting.